sql/database.sql

```diff
@@ -9638,3 +9638,16 @@ CREATE TABLE `codes_history` (
   `update_by` varchar(255),
    PRIMARY KEY (`log_id`)
 ) ENGINE=InnoDB;
+
+-- Table to store multiple db connection details
+DROP TABLE IF EXISTS `zf_multiple_db`;
+CREATE TABLE `zf_multiple_db` (
```

Comment:
lets keeps this generic and call it multiple_db
(since it may get expanded to use outside of zend in the future)